app/views/pages/home.html.erb

```diff
@@ -10,59 +11,23 @@
 
 <div class="row">
   <div class="col-md-7">
-
     <div class="panel panel-default">
       <div class="panel-body">
-        <%= link_to 'New Post', new_post_path, class: 'btn btn-primary pull-right' %>
-        <%= link_to 'Tags', tags_path, class: 'btn btn-primary pull-right tag-btn' %>
-        <h2>Latest Posts</h2>
+        <%= link_to "New Post", new_post_path, class: 'btn btn-primary pull-right' %>
+        <%= link_to "Tags", tags_path, class: 'btn btn-primary pull-right tag-btn' %>
+        <h2><%= "Latest Post" %></h2>
```

Comment:
This does not need to be rendered by ERB. It can be plain HTML: `<h2>Latest Posts</h2>`